### DISCUSSION TITLE
jekyll build failure after dec 2021

### DISCUSSION THREAD

**Discussion by aaron-hd** [mmistakes/minimal-mistakes#3320] [Q&A]
We use minimal-mistakes for a public website via git (https://github.com/aaron-hd/as-website).
Today we committed new posts but the build failed. Reverting the git to the last working commit (dec-2021) did not change the build failure. Hence, the same commit which was successfully built in dec-2021 now fails to build.

Did anyone have similar issues?
Does anyone know how to solve this issue?

Many thanks in advance.

**Comment by antenore** [comment]
Maybe you can have a look at the build log and figure out what's happening.
There haven't been any new releases and/or commit since October 19, therefore it's probably due to some gem updates, or other issues in your CI.
